src/server/game/Entities/Creature/TemporarySummon.h

```diff
@@ -59,8 +59,22 @@ class TC_GAME_API Minion : public TempSummon
         Unit* GetOwner() const { return m_owner; }
         float GetFollowAngle() const override { return m_followAngle; }
         void SetFollowAngle(float angle) { m_followAngle = angle; }
-        bool IsPetGhoul() const {return GetEntry() == 26125;} // Ghoul may be guardian or pet
-        bool IsSpiritWolf() const {return GetEntry() == 29264;} // Spirit wolf from feral spirits
+
+        // Warlock pets
+        bool IsPetImp() const { return GetEntry() == 416; }
```

Comment:
could be a good idea to remove these magic numbers

Reply:
`NPC_IMP             = 416` is already enumerated in this file.